pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -121,11 +166,31 @@ func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Contai
 	}
 
 	machineState := s.GetMachineState()
+	reusable := p.memoryToReuse[string(pod.UID)]
+	// requested memory - reusable 
+	requestedAbsolute := make(map[v1.ResourceName]uint64)
+
+	for resourceName, resourceSize := range requestedResources {
+		//copy a map
+		requestedAbsolute[resourceName] = resourceSize
+
+		if _, ok := reusable[resourceName]; !ok {
+			continue
+		} 
+
+		reusableSize := reusable[resourceName].Size
```

Comment:
like you wrote above we should be sure that `reusable[resourceName].Size` > 0

Reply:
This is checked now in `updateMemoryToReuse()`. Should it be checked here too?